vue-napredno/src/views/admin/users/UserListView.vue

```diff
@@ -13,30 +13,43 @@
           :server-items-length="state.totalItems"
           @pagination="getProductList"
           @click:row="openProfile"
```

Comment:
Još uvijek je `@click:row` ovdje, ne treba ga definirati ako već postoji gumb za to unutar overridanog rowa.